core/performance_test/peakflops/01_cpp_simple.cpp

```diff
@@ -118,7 +118,7 @@ struct data * axpy_10(struct data *__restrict__ a, struct data *__restrict__ x0,
 
 
 TEST(cpp, simple) {
-  long n = static_cast<long>(2e6);
+  long n = static_cast<long>(2e6); //use a higher number e.g. 2e8 for skylake 
```

Comment:
Why are we hard-coding this? Could we make this into a CMake option? Have it be `2e8` by default and use `2e6` for the CI.

Reply:
@junghans Sounds like a good idea. I am not sure how to do that. Could you help? Thanks.

Reply:
You can just copy what @rfbird did for `Cabana_PERFORMANCE_ERROR_MARGIN`:
https://github.com/ECP-copa/Cabana/blob/master/core/performance_test/peakflops/CMakeLists.txt#L17-L22

Reply:
I can do this for you Gy

Reply:
I pushed something to your branch that does it (https://github.com/guangyechen/Cabana/commit/b15345aadac0082eae3e3171abb69112b82d8cda), please take a look

I think we should also consider adding prints to feed back to the user so they can see if their attempt at changing it stuck -- what do you think? (print n and flops etc)